07_deploy_masters.sh

```diff
@@ -73,7 +73,7 @@ wait_for_bootstrap_event
 
 # disable NoSchedule taints for masters until we have workers deployed
 for num in 0 1 2; do
-  oc adm taint nodes master-${num} node-role.kubernetes.io/master:NoSchedule-
+#  oc adm taint nodes master-${num} node-role.kubernetes.io/master:NoSchedule-
```

Comment:
Oh one more - should this be commented out?

Reply:
Doh, no - it was failing in my environment which I think was a spurious error, thanks, now fixed!